draft-ietf-rats-architecture.md

```diff
@@ -1052,7 +1052,7 @@ or might be defined relative to some other timestamp or timeticks counter.
 | VG | Value generation            | A value to appear in a Claim was created.
 | AA | Attester awareness          | An Attesting Environment starts to be aware of a new/changed Claim value.
 | CC | Claim Collection            | An Attesting Environment collects a new/changed Claim value to appear in Evidence.
-| HD | Handle distribution         | A centrally generated identifier for time-bound recentness across a domain of devices is successfully distributed to Attesters.
+| HD | Handle distributed         | A centrally generated handle for time-bound freshness is successfully received by all entities in a domain.
```

Comment:
Whose clock is addressed by time HD?

Reply:
Handles a created at time(X) and received at time(Y). time(X) is typically easily obtainable and non-ambiguous, but not very useful. time(Y) introduces a variance of uncertainty across a certain time window. Effectively, that keeps them at the same point of time in low resolution timestamps and, analogously, not, if the resolution of timestamps is fine-grained enough. As we successful avoided to talk about timestamp resolution (which is essential to remote attestation, but not to this document), we'd like to keep it that way. Therefore, the choice if it is handle generation, handle emission, or handle reception is up to us here. I'd preferred distribution (encompassing the latency by referencing the start of distribution -> emission), but I can see how the perception is always falling back to "local clocks" even if they might not be there.

Some high level context: in the end, this is for us to decide. Handle Distribution is used a lot in constraint node environments.
Sometimes things have clocks, sometimes they don't.
Sometimes you want to use a trustable central times authority, sometimes you don't.
Sometimes you can infer a time when values were generated, sometimes you can't.
Depicting permutations of combinations of all existing factors would result in 32+ examples. So that did not seem reasonable.

Hence, example 4 is composite of the most typical constraints that you encounter in IoT:

* sometimes no clocks
* sometimes no central time
* sometimes no capability for value monitoring

My personal goal is to capture these three essential items into example 4. Any help is welcome!